sdk/fixtures/src/main/java/com/klaviyo/fixtures/MockIntent.kt

```diff
@@ -0,0 +1,87 @@
+package com.klaviyo.fixtures
+
+import android.annotation.SuppressLint
+import android.content.Intent
+import android.net.Uri
+import android.os.Bundle
+import io.mockk.CapturingSlot
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.mockkConstructor
+import io.mockk.slot
+
+/**
+ * Data class to hold all Intent capturing slots for easy access and destructuring
+ */
+data class MockIntent(
```

Comment:
I extracted this test utility for mocking Intent across all tests. Makes it easier to get tests involving intents off the ground, particularly if you need to verify how intent properties are set/mutated.